static-assets/components/cstudio-forms/controls/rte.js

```diff
@@ -963,14 +963,31 @@ CStudioAuthoring.Module.requireModule(
           }
         },
 
+        /**
+         * Check input with space only
+         */
+        _isInputSpace: function (obj) {
+          if (!obj.validateTextArea) {
+            obj.validateTextArea = document.createElement('textarea');
+          }
+          obj.validateTextArea.innerHTML = obj.value;
+
+          if (!obj.validateDiv) {
+            obj.validateDiv = document.createElement('div');
+          }
+          obj.validateDiv.innerHTML = obj.validateTextArea.value;
+
+          return obj.validateDiv.innerText.trim() == '';
+        },
+
```

Comment:
See comment above (move to comment api)

Reply:
Thanks @rart for the suggestion. I fixed to move it to common API. Kindly check again